lib/ios/UINavigationBar+utils.h

```diff
@@ -0,0 +1,9 @@
+#import <UIKit/UIKit.h>
+
+@interface UINavigationBar (utils)
+
+- (void)rnn_setBackgroundColor:(UIColor *)color;
```

Comment:
Why do we append the `rnn_` prefix to these methods? Can we avoid prefixing methods?